mx.jvmci/mx_jvmci.py

```diff
@@ -1463,6 +1463,8 @@ def _jvmci_versions_in_current_branch():
             m = jvmci_tag_re.match(decoration)
             if m:
                 versions.append(tuple([int(g) for g in m.groups()]))
+    if not versions:
+        raise Exception('No JVMCI versions found. If this is a shallow clone of the git repo, it may need to be unshallowed and load tags with `git fetch --unshallow --tags`')
```

Comment:
Looks good except that `raise Exception` should be `mx.abort`.

Reply:
Thanks. Changed, and force pushed.